python/lsst/pipe/analysis/plotUtils.py

```diff
@@ -295,6 +321,33 @@ def bboxToRaDec(bbox, wcs):
     ra, dec = zip(*corners)
     return ra, dec
 
+def getRaDecMinMaxPatchList(patchList, tractInfo, pad=0, nDecimals=4, raMin=360.0, raMax=0.0,
+                            decMin=90.0, decMax=-90.0):
+    """Find the max and min RA and DEC (deg) boundaries encompased in the patchList
+
+    @param[in] patchList  List of patch IDs
+    @param[in] tractInfo  Tract information associated with the patches in patchList
+    @param[in] pad        Pad the boundary by pad degrees
+    @param[in] nDecimals  Round coordinates to this number of decimal places
+    @param[in] raMin      Initiate minimum RA determination at raMin (deg)
+    @param[in] raMax      Initiate maximum RA determination at raMax (deg)
+    @param[in] decMin     Initiate minimum DEC determination at decMin (deg)
+    @param[in] decMax     Initiate maximum DEC determination at decMax (deg)
+    """
+    for ip, patch in enumerate(tractInfo):
```

Comment:
Why not call [`TractInfo.findPatchList`](https://github.com/lsst/skymap/blob/master/python/lsst/skymap/tractInfo.py#L178)?